schema/extensions/skosext.ttl

```diff
@@ -0,0 +1,23 @@
+# baseURI: http://qudt.org/2.1/schema/extensions/skosext
```

Comment:
propose http://qudt.org/2.1/schema/qudt/extensions/skos

rationale:

same as qudt schema base uri up to /extensions/skos

Reply:
The reason I didn't call it skos is because the intuitive prefix for that would then be "skos", but that collides with the prefix for the real skos. Of course, we can still define the prefix to be skosext anyway, but it kind of breaks with convention.
Thoughts?

Reply:
Agreed to the URI being http://qudt.org/2.1/schema/qudt/extensions/skos
If we choose to define a prefix, it could be something like qudt.skos or qudt-skos.